### ISSUE TITLE
[1849] RSA shouldn't close when reopened RSA corporation buys a train

### ISSUE THREAD

**Issue by crericha** [tobymao/18xx#6049]
1. RSA corporation closed without buying a train
2. RSA corporation is reopened in the next SR
3. RSA corporation buys a train
4. RSA closes, but it shouldn't

In AAG rulebook "If the associated corporation closes before buying a train, the P5 Reale Societa dAffari remains open until all private companes are closed at the start of Phase 12".

https://18xx.games/game/49896?action=187

**Comment by slapphappyjazz** [comment]
I think this needs a rule adjudication.  The rules also say that P5 closes when the RSA corporation buys a train. My interpretation of the rule quoted is that it is intended to cover the case when the RSA corp closes and doesn’t re-enter play; I.e. the case you quote wasn’t properly covered by the rules.

**Comment by scottredracecar** [comment]
Okay, will look into this.

**Comment by crericha** [comment]
I believe we already have a ruling from the designer. I'll try to find it
again.

**Comment by scottredracecar** [comment]
I don't think we will find something from the designer on this one, but I take back the following if we do and it conflicts.

In discussing briefly with Jason (primary writer on the new version), I think we added the rule because the case was not covered and it was consistent with the previous version's rules as written. 

The new rule:
If the associated corporation closes before buying
a train, the P5 Reale Società d’Affari remains open until all private
companies are closed at the start of Phase 12."

In discussing today, the rationale is that a company closure is a "hard reset" on cash/trains/privates and it is consistent to also sever the connection to the RSA in our opinion.

So in conclusion, bug confirmed. But if whoever is going to fix this needs Vellani's blessing, then we will need to bug him about this thing that he probably hasn't considered and doesn't feel strongly about (my impression from feedback received in the past).

**Comment by dfannius** [comment]
I'm happy to consider the current behavior broken without confirmation from Vellani (and it's the ruling I would make as well); the discussion above is enough for me.
